### PR TITLE
refactor (pong): no more threads

### PR DESCRIPTION
**Pull request by Stte**
Removed threads and refactored the code to be easier to follow

[How to test server-side pong with Postman](https://transcendance.atlassian.net/l/cp/1mm5SHBa)

### CODE REVIEW

File: backend/django_backend/pong/consumers.py [hunk 1]
```diff
@@ -34,51 +33,46 @@ async def connect(self):
         if not self.pong_game.room_group_name:
             self.pong_game.add_room_group_name(self.room_group_name)
 
+        # Add player to game
         if not self.pong_game.player1:
             self.player = Player(1, "Player 1")
             self.pong_game.add_player(self.player)
-            self.thread["player1"] = True
         elif not self.pong_game.player2:
```
Comment (Stte) [inline backend/django_backend/pong/consumers.py:40]: For RushMaverick: You can remove the line 40 (elif) when you are working on your threejs ;)

**Review by jestebanpelaez18 — APPROVED** [review]
nice !

File: backend/django_backend/pong/consumers.py (hunk 1 above)
Comment (RushMaverick) [reply backend/django_backend/pong/consumers.py:40]: Is this because we are not using a player2 in that moment?

**Review by RushMaverick — APPROVED** [review]
If I understood correctly, instead of threads we are now using an array of games? 
Also, please assign people as reviewers for the next PR 💪

Good work, I'll test it out.

File: backend/django_backend/pong/consumers.py (hunk 1 above)
Comment (Stte) [reply backend/django_backend/pong/consumers.py:40]: Both players are working but then you don't have to open two browsers to test. Plus currently both players have to disconnect before you can start a new game with the same name.

**Pull request merged**